app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
     # write your code here
-    pass
+    print(len(people))
+    person_list = []
+    for person in people:
+        if "wife" in person and person['wife'] != None:
+            name = Person(person["name"], person["age"])
+            name.wife = person['wife']
+
+            person_list.append(name)
```

Comment:
you can write "append" outside if-else statement in order not to repeat it three times

Reply:
the code won't work in this way

Reply:
you don't need continue
del it and write "append" outside if-else statement in order not to repeat it three times
and your cod will work correct